static_src/components/app_list.jsx

```diff
@@ -43,9 +44,9 @@ export default class AppList extends React.Component {
   }
 
   appUrl(app) {
-    return `/#/org/${ this.state.currentOrgGuid }
-      /spaces/${ this.state.currentSpaceGuid }
-      /apps/${ app.guid }`;
+    return dedent`/#/org/${ this.state.currentOrgGuid }
```

Comment:
Nice! I know the bug that this was causing but definitely didn't guess that it was related to multi-line template strings